src/app/profile/overview/connected-accounts/connected-accounts.component.html

```diff
@@ -41,10 +41,20 @@ <h2 class="card-pf-title">
                         <span>
                           Connected
                         </span>
+                        <span>
+                          <button class="btn btn-lg btn-default" id="gh_relink"
+                          (click)="connectGitHub()">Reconnect</button>
+                          <button class="btn btn-lg btn-default" id="gh_unlink"
+                          (click)="disconnectGitHub()">Disconnect</button>
+                        </span>
                       </ng-template>
                       <ng-template #ghUnlinkedUsername>
                         <span class="account-disconnected">
```

Comment:
same here - add class `margin-right-15` to apply a space between this text and the button